.github/workflows/oidc-test.yml

```diff
@@ -33,42 +33,32 @@ jobs:
       # Generating a unique name for the Integration Configuration that will be created in the following step
       - name: Generate unique OIDC config name
         shell: bash
-        run: echo "OIDC_CONFIG_NAME=oidc-integration-test-config-$(date +%Y%m%d%H%M%S)" >> $GITHUB_ENV
+        run: echo "OIDC_PROVIDER_NAME=oidc-integration-test-provider-$(date +%Y%m%d%H%M%S)" >> $GITHUB_ENV
 
       - name: Create OpenID Connect integration
         shell: bash
         run: |
           curl -X POST "${{ secrets.JFROG_PLATFORM_URL }}/access/api/v1/oidc" -H "Content-Type: application/json" -H "Authorization: Bearer ${{ secrets.JFROG_PLATFORM_RT_TOKEN }}" -d '{
-          "name": "${{ env.OIDC_CONFIG_NAME }}",
+          "name": "${{ env.OIDC_PROVIDER_NAME }}",
           "issuer_url": "https://token.actions.githubusercontent.com/",
           "provider_type": "GitHub",
           "description": "This is a test configuration created for OIDC-Access integration test" }'
 
-      - name: Set subject
```

Comment:
Can you please explain to me what does this step suppose to do?

Reply:
It used to set the "subject" key, but dealing with the "repository" is simpler, so I took it out.